### ISSUE TITLE
[1.14.4] [CIT] Transparent armor flickering between opaque and transparent

### ISSUE THREAD

**Issue by TBlazeWarriorT** [sp614x/optifine#4081]
## Description of Issue
Transparent/translucent armor flickers between opaque and transparent depending on view angle, looks semi-random, should always display transparency properly. Related to https://github.com/sp614x/optifine/issues/3209

## Steps to Reproduce
1. Get transparent armor textures. I found this bug using CIT. Example files for testing, if needed:

crystal.properties: [crystal.properties](https://github.com/sp614x/optifine/files/4768649/crystal.txt)
blank.png
![blank](https://user-images.githubusercontent.com/25572454/84462811-557a7b80-ac46-11ea-9de9-919c4288c5b9.png)
crystal_layer_1.png
![crystal_layer_1](https://user-images.githubusercontent.com/25572454/84462805-54494e80-ac46-11ea-823b-cdf901c0d977.png)
crystal_layer_2.png
![crystal_layer_2](https://user-images.githubusercontent.com/25572454/84462808-557a7b80-ac46-11ea-9a14-2861ee86e4de.png)

## OptiFine Version
1.14.4 HD U F5

## F3 Debug Screenshot
![image](https://user-images.githubusercontent.com/25572454/84462990-ca4db580-ac46-11ea-99e2-aacb0b329874.png)

## Prior Testing
- Does this happen in Vanilla without OptiFine?
Can't test properly (vanilla has no CIT to test)

## Additional Information
Still happens if using internal or any shaders. Seems to be unrelated to shaders.
Still happens after toggling settings suck as fast render on/off. Seems to be unrelated to settings.

**Comment by TBlazeWarriorT** [comment]
Can be seen as a video here:
https://youtu.be/Zk-cb9fyxCI

**Comment by Vitalitykai** [comment]
Did this ever get fixed?

**Comment by TBlazeWarriorT** [comment]
> Did this ever get fixed?

no idea, I migrated to CIT Resewn a few years ago

**Comment by Vitalitykai** [comment]
so it works on CIT Resewn and if so could u tell me how u did it?

**Comment by TBlazeWarriorT** [comment]
> so it works on CIT Resewn and if so could u tell me how u did it?

Nope, it seems fully opaque on the model. Base 1.20.1 Minecraft doesn't seem to support transparent armor, and CIT Resewn does not change it, for better or for worse. CIT Resewn or any other mod would need to and translucency support, then it would probably work. I don't think translucent armor is viable to this day, other than the half-working one described on the original issue several years ago.

**Comment by Vitalitykai** [comment]
damn

**Comment by TBlazeWarriorT** [comment]
> damn

You can test it on latest OptiFine to be sure and update the issue with the current status if you'd like, to see if maybe you're really lucky and it magically works

**Comment by Vitalitykai** [comment]
Does not work at all on either CIT Resewn, Optifine or Vanilla
